ticdc/ticdc-changefeed-config.md

```diff
@@ -37,6 +37,11 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
 This section introduces the configuration of a replication task.
 
 ```toml
+# The memory limit the changefeed can used in the caputre server by the sink 
+# manager. If exceed, the overused part will be recycled by the go 
+# runtime prioritily. The default value is 1GB 
```

Comment:
by the go runtime priority 这句话看不太懂，需要优化下。@charleszheng44
```suggestion
# Specifies the memory quota in bytes. It can be used in the caputre server by the sink manager.
# If the value is exceeded, the overused part will be recycled by the go runtime priority.
# The default value is `1073741824` (1 GB). 
```

Reply:
rephrased